basic_bootloader/src/bootloader/constants.rs

```diff
@@ -62,13 +62,15 @@ pub const ERC20_ALLOWANCE_SELECTOR: &[u8] = &[0xdd, 0x62, 0xed, 0x3e];
 // 0x095ea7b3
 pub const ERC20_APPROVE_SELECTOR: &[u8] = &[0x09, 0x5e, 0xa7, 0xb3];
 
-// Value taken from system-contracts, to adjust.
-pub const L1_TX_INTRINSIC_L2_GAS: u64 = 11000;
+// Just for EVM compatibility.
+pub const L1_TX_INTRINSIC_L2_GAS: u64 = 21_000;
 
-// Includes storing the l1 tx log.
-pub const L1_TX_INTRINSIC_NATIVE_COST: u64 = 10_000;
+// Includes:
+//  - Storing and hashing the l1 tx log.
+//  - Transferring fee to coinbase.
+pub const L1_TX_INTRINSIC_NATIVE_COST: u64 = 70_000;
```

Comment:
@antoniolocascio it doesn't include tx hashing, right? Ideally we should cover all the things we are doing during validation with intrinsic cost

Reply:
Merging it. We can make additional changes in a separate PR